src/Airspace/AirspaceParser.cpp

```diff
@@ -695,7 +708,8 @@ ParseLine(Airspaces &airspace_database, unsigned line_number,
     case 'Y':
     case 'y':
       if (input.SkipWhitespace())
-        temp_area.astype = string_converter.Convert(input.c_str());
+        if (IsICAOClassOrUnclassified(temp_area.asclass))
+          temp_area.astype = ParseClass(input.c_str());
```

Comment:
I don't like that this (ab)uses the `ParseClass()` function for both `AC` and `AY`. These are different records, and have different values.